articles/openshift/openshift-faq.md

```diff
@@ -144,7 +144,7 @@ Caution must be used when using specific labels:
 
 ## What is the maximum number of pods in an ARO cluster?  What is the maximum number of pods per node in ARO?
 
-Refer to [upstream OpenShift docs](https://docs.openshift.com/container-platform/3.11/scaling_performance/cluster_limits.html#scaling-performance-current-cluster-limits) for more details. Red Hat OpenShift 3.11 has a 250-pod/node limit, whereas [ARO has a 20-compute node limit](https://docs.microsoft.com/azure/openshift/openshift-faq#what-cluster-operations-are-available), so that caps the maximum number of pods supported in an ARO cluster to 250*20 = 5000.
+Refer to [upstream OpenShift docs](https://docs.openshift.com/container-platform/3.11/scaling_performance/cluster_limits.html#scaling-performance-current-cluster-limits) for more details. Red Hat OpenShift 3.11 has a 50-pod/node limit, whereas [ARO has a 20-compute node limit](https://docs.microsoft.com/azure/openshift/openshift-faq#what-cluster-operations-are-available), so that caps the maximum number of pods supported in an ARO cluster to 250*20 = 5000.
```

Comment:
This link doesn't exist anymore and linking to this documentation that once existed showed the limits of the  OpenShift Container Platform and not Azure Red Hat OpenShift with having a limit of 250 pods per node.  Rewording this to show that Azure Red Hat OpenShift only allows a capacity of 50 pods per node as a capacity and not link to the documentation will rectify the discrepancy. @sakthi-vetrivel